include/directx/d3dx12.h

```diff
@@ -4037,6 +4038,421 @@ class CD3DX12_NODE_MASK_SUBOBJECT
     D3D12_NODE_MASK m_Desc;
 };
 
+//------------------------------------------------------------------------------------------------
+// Peter: Implementation for the new CheckFeatureSupport API

```

Comment:
Best practice is not to include author names in comments, as this won't mean anything to people not on the team and soon becomes obsolete as people move around and teams change.  The author can always be found in git history if needed.

Reply:
Resolved. Removed my name.